### DISCUSSION TITLE
Why doesn't katana work at the office but does at home (out of curiosity)

### DISCUSSION THREAD

**Discussion by TazDevl74** [projectdiscovery/katana#193] [Q&A]
Did some testing with Katana on the Corp and Guest network. For some reason I did not get any results. First thought was that it could be an incompatibility with my ARM64 hardware in one or more of the required apps. 

So I downloaded go.mod (dependency list) to try again at home. To my surprise the queries that did nothing, worked on my private network.

What could cause Katana not to work? Should I rate limit for instane?

**Comment by sullo** [comment]
Most often I've seen problems like this, it is a corporate proxy in the way. Could that be the case here? You might need to set your upstream proxy to the corporate one, and authenticate if necessary. Does it work against an internal target?

**Comment by TazDevl74** [comment]
Nope, don't have a proxy anymore on both networks

**Reply by sullo** [reply]
How about on internal targets?
